### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by DevIgork**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,8 @@
+MYSQLDB_ROOT_PASSWORD=12345
```
Comment (annton-shakur) [inline .env:1]: .env file should not be pushed to the main branch, consider adding it to .gitignore

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    build: .
+    env_file: ./.env
+    ports:
+      - $SPRING_LOCAL_PORT:$SPRING_DOCKER_PORT
+      - $DEBUG_PORT:$DEBUG_PORT
+    environment:
+      SPRING_APPLICATION_JSON: '{
+      "spring.datasource.url" : "jdbc:mysql://mysqldb:$MYSQLDB_DOCKER_PORT/$MYSQLDB_DATABASE?useSSL=false
+      "spring.datasource.username" : "root"
+      "spring.datasource.password" : "12345"
```
Comment (annton-shakur) [inline docker-compose.yml]: you can use env variables here instead

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      "spring.datasource.driver-class-name" : "com.mysql.cj.jdbc.Driver"
+      "spring.jpa.hibernate.ddl-auto" : "validate"
+      "spring.jpa.show-sql" : "true"
+      }'
+      JAVA_TOOL_OPTIONS : "-agentlib:jdwp=transport=dt_socker,server=y,suspend=n,address=*:5005"
```
Comment (annton-shakur) [inline docker-compose.yml]: pls add an empty line here

**Review by annton-shakur — CHANGES_REQUESTED** [review]
Good job! pls pay attention to my comments. Also, all the changes that are not related to docker must be added in a separate PR instead of this one

File: docker-compose.yml (hunk 2 above)
Comment (vovasalyha) [reply docker-compose.yml]: Agree, use env variables

File: docker-compose.yml (hunk 3 above)
Comment (vovasalyha) [reply docker-compose.yml]: Agree

File: src/main/java/com/example/bookstore/controller/OrderController.java [hunk 4]
```diff
@@ -52,10 +52,10 @@ public List<OrderDto> getAll(Authentication authentication) {
     @Operation(summary = "Patch status", description = "change status of order")
     @PreAuthorize("hasRole('ROLE_ADMIN')")
     @PatchMapping("{id}")
-    public OrderDto UpdateStatus(
+    public OrderDto updateStatus(
```
Comment (vovasalyha) [inline src/main/java/com/example/bookstore/controller/OrderController.java:55]: Don't push unrelated changes to PR

File: .env (hunk 1 above)
Comment (vovasalyha) [reply .env:1]: Agree, but in this project it can be for learning purposes, so not a big deal. The most important thing is to not push any real credentials here, but some dummy ones.

**Review by vovasalyha — APPROVED** [review]
Looks good, but please take a look on your colleague's comments.

File: src/main/java/com/example/bookstore/controller/OrderController.java (hunk 4 above)
Comment (d1sam) [reply src/main/java/com/example/bookstore/controller/OrderController.java:55]: still exists

File: src/main/java/com/example/bookstore/dto/order/OrderDto.java
Comment (d1sam) [inline src/main/java/com/example/bookstore/dto/order/OrderDto.java:1]: unrelated change to pr

File: src/main/java/com/example/bookstore/model/Order.java
Comment (d1sam) [inline src/main/java/com/example/bookstore/model/Order.java:1]: unrelated change to pr

File: src/main/java/com/example/bookstore/repository/CartItemRepository.java
Comment (d1sam) [inline src/main/java/com/example/bookstore/repository/CartItemRepository.java:1]: unrelated change to pr

File: src/main/java/com/example/bookstore/repository/OrderItemRepository.java
Comment (d1sam) [inline src/main/java/com/example/bookstore/repository/OrderItemRepository.java:1]: unrelated change to pr

File: src/main/java/com/example/bookstore/repository/OrderRepository.java
Comment (d1sam) [inline src/main/java/com/example/bookstore/repository/OrderRepository.java:1]: unrelated change to pr

File: src/main/java/com/example/bookstore/service/OrderItemService.java
Comment (d1sam) [inline src/main/java/com/example/bookstore/service/OrderItemService.java:1]: unrelated change to pr

File: src/main/java/com/example/bookstore/service/OrderService.java
Comment (d1sam) [inline src/main/java/com/example/bookstore/service/OrderService.java:1]: unrelated change to pr

File: src/main/java/com/example/bookstore/service/impl/OrderItemServiceImpl.java
Comment (d1sam) [inline src/main/java/com/example/bookstore/service/impl/OrderItemServiceImpl.java:1]: unrelated change to pr

File: src/main/java/com/example/bookstore/service/impl/OrderServiceImpl.java
Comment (d1sam) [inline src/main/java/com/example/bookstore/service/impl/OrderServiceImpl.java:1]: unrelated change to pr

**Review by d1sam — CHANGES_REQUESTED** [review]
please remove all unrelated changes from this pr and make them in another pr.
